src/stores/apps.store.ts

```diff
@@ -34,6 +35,14 @@ type AppId = number
 
 type WalletModeKind = WalletMode.NoUI | WalletMode.UI
 
+type Delegate = {
+  id: number | string
+  name: string
+  address: string
+  permissions: string[]
```

Comment:
If these are going to a limited set of string, we should use a union type: 

```ts
type DelegatePermission: 'Download' | 'Reshare'

type Delegate = {
  ...
  permissions: DelegatePermission[]
}
```

Reply:
Resolved here https://github.com/arcana-network/developer-dashboard/pull/144/commits/0bc873dc5c776a300fd66a288be2a8b8ed676090